common/main/hi_Latn.xml

```diff
@@ -5361,58 +5361,58 @@ annotations.
 			</decimalFormatLength>
 			<decimalFormatLength type="long">
 				<decimalFormat>
```

Comment:
do these paths under "long" count as "compact"? I imagine they do given the ticket name... I'm ignorant about what "compact" means here...

Reply:
that's a good question.  I think they are still compact in that they aren't spelled out.   (`123 thousand`,  not `one hundred twenty three thousand`) .  Maybe get @macchiati  and @AEApple comment?

Reply:
Yes, it's that they aren't spelled out as a number so 1 million would be the compact number form vs. 1,000,000.